.travis.yml

```diff
@@ -1,22 +1,43 @@
+  - secure: tzMJDFpqOq7xeUmYzLUHkvgYzXCJvnPbtl8GKkPfKDOD5FLwfLPYkJZ6ElPPei2IFxAoKmBQEwBWxb64KmYC9RwLANovF7JW6B40B/yi0wQrz9u0Mzx5JG9lVoD6EZggaJtzvRV6Gm692BzDaxRMpEUhcm1tCCY1RmTHAX+VvlNbK+pljowJPPLdXQAiFW1Sy4WI0J8CuCQSnp2x8XRXd2MVAHbX3CKB1B31OV3nTsLxp12WRJDLxHYM3lUddDMh+PlfCtAodMY+gaO2mVsFWEDD3bhTVmgpfh4d2m7lLyeExyTfrRrd/1k4OKP2welKlXlZ0vPU5BJJgyz2OD5Rsgf3f1BaJ0a/YtHbMnV5sJGi59OBkyA406a/lE8ZJ0kuD+/ofe8mH02T6Zcv+M6fZLY85+MDyzch4jDpbEGmiOPSHTddDBv/ImUeNeYacgagg3WOfCktl1QzRvt22H90AHgnndVIsFCZNxlQgipCYpeteBIZxPuvMZVxhnrQ9TxTVP4pgT/RS09xZrmkTsnyDAkgeuJQMCvBS0fSqRC13WGQjiEPN7IWgX5RaZ2neO/HeYZhfoelF/Xbm7qj1hljlcyFuuGDmasfEtmykHZ5Zl+zylu6QfgGvTwUWeil9uQctHG5+tWnHJYpBJPuslWFD1uUMIUp8VDpYZcTqpzBw2w=
 notifications:
   slack:
     secure: xsQM9c0gQYB4HK9ugFF5jdE/1sHcAytF7bB9sVtC4jrDBxLaHFtQYrdijzbf3yn9kKUbcOtESov1ary4e9NcF3751tbt0hD6b6QT2Dbx6ZcWrKB6MuCY/ady/T4KvwVlPOwsBk5QXyG2anRxaV1nqPu4/eUPVHe+RBNDkNzU88WcYx8BsUCsWIG4u9uWs8WJj4KVbgGNMoi8AflpB/YeQOYY0TOmVP84/QM4w8EwssE8fHGpNsq+Ad1iGyKLF1HSon9dAIT4QEx4DnJ6N8hT+oGWjIGaiS6MsZHH6w0mL2D0mBxTHaoDoOqjaZ+rDsbSDhM6c+WpLTde1nK0sbcgbvWDGD+g5fYmX43QNv75++7CkO0+GTuNCKF9vnW/P4eaqNjPvpR4vBbGjWw8cQsPjhinLf6ZRRdURYGEAmKfX3c2SNprEi5bto4kI4rmWbrR1cs3ny7esz/lKg6RsqWrTvDHeVsmIIXDSH+gHFtHK0pjlvrgLKwZUNMwKUh21wGY/xgXVNk4r++18S3FyitWBkEP2HRf8eWGDgqWCXSDqMOj1OKeyXAEUiXeSB7m/qPIrVEqqXNc6IRoxETBlWZg4xf5ZZJMCPJu65e4D7GL9TMGNmhTC6Er8xAZB4NN/mhikFC5mNGhxaMOJ4FnCcy2tn/nZWrcL4Gqb9zyNcEk1ys=
   email:
-    - wijimec635@poetred.com
+  - wijimec635@poetred.com
 script:
-- echo "Build unit tests"
-- docker build --target unittests --tag todo-app:unittests .
-- docker run todo-app:unittests
-- echo "integration tests"
-- docker build --target integrationtests --tag todo-app:integrationtests .
-- docker run --env-file ./.env.test todo-app:integrationtests
-- echo "endtoend tests"
-- docker build --target endtoendtests --tag todo-app:endtoendtests .
-- docker run -e APP_API_KEY=$APP_API_KEY -e APP_TOKEN=$APP_TOKEN todo-app:endtoendtests
+  - echo "Build unit tests"
+  - docker build --target unittests --tag todo-app:unittests .
+  - docker run todo-app:unittests
+  - echo "integration tests"
+  - docker build --target integrationtests --tag todo-app:integrationtests .
+  - docker run --env-file ./.env.test todo-app:integrationtests
+  - echo "endtoend tests"
+  - docker build --target endtoendtests --tag todo-app:endtoendtests .
+  - docker run -e APP_API_KEY=$APP_API_KEY -e APP_TOKEN=$APP_TOKEN todo-app:endtoendtests
+after_success:
```

Comment:
Note that this will mean that every successful test run updates your 'prod' image - in a real project we'd likely only want that to happen when we're actually ready to deploy (e.g. after it's been merged, not just when it's up for review). Switching to the `before_deploy` lifecycle stage might make more sense